### ISSUE TITLE
[Common Releases] Delete release_notes

### ISSUE THREAD

**Issue by pgoyal01** [cntt-n/CNTT#2701]
Anuket Project has a Release Process and Release artifacts defined in the [Releases Home](https://wiki.anuket.io/display/HOME/Releases+Home), and hence suggest that we delete the common/release_notes.

Please comment if you agree.

**Comment by CsatariGergely** [comment]
I agree, we should have a common release notes for Anuket, maintained in the docs repo.
